ngzip.gemspec

```diff
@@ -2,7 +2,7 @@
 
 lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'english'
+require 'English'
```

Comment:
Thanks for this - I'm using macOS and this did obviously work on the case-insensitive file system ..